### ISSUE TITLE
ComboBox: Can't pick values from the control on Edge with Narrator ON

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#5289]
This issue can also be observed on https://developer.microsoft.com/en-us/fabric#/components/ComboBox on Edge with narrator on. In this case the control doesn't allow to choose values using the keyboard arrow keys. Hence it becomes nearly impossible for a person with accessibility needs to navigate from oe option to another.

**Comment by jspurlin** [comment]
@ssingh28 , is this specific to scan mode? If so it sounds like it may be a bug in Narrator

**Comment by ssingh28** [comment]
@jspurlin @dzearing : This issue is reproducible with keyboard navigation on Edge with narrator ON with Scan mode off (Caps Lock + Space Bar) and the narrator does announce scan off. Please let me know if there is anything which is missing here.

**Comment by jspurlin** [comment]
Can you go into more details on your repro steps? I'm able to interact with comboBoxes in Edge with Narrator on (and scan mode off) as expected.

**Comment by mikewheaton** [comment]
@ssingh28 Are you still seeing this issue?

**Comment by JasonGore** [comment]
As other comments have indicated, I am able to select items so I'm going to mark this as can't repro. We do have another issue (#6920) that highlights some other keyboard navigation issues, so I will close this one for now.
